model/Core/Classes/X-Collection.md

```diff
@@ -4,14 +4,22 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Assembles a collection of Elements that does not contain any other X Collection Elements.
+A collection of Elements that respresents a serialization of SPDX data.
 
 ## Description
 
-An X collection Element is a collection of Elements that does not contain any other X Collection Elements.
+The X-Collection provides a convenient way to store information about SPDX data which has been serialized as a complete
+unit (e.g., all SPDX data within a single JSON-LD file).
+Information we wish to preserve about the serialization itself are store as properties of this class.
 
-In this way it can be thought of as an outer shell collection of SPDX content without 
-self-recursion that can be used as a content aggregation target for serialization.
+NOTE: This class is not intended to be serialized itself.
```

Comment:
But no one prevents it from being serialized. It just has no implication to the serialized blob it contains.

Reply:
@maxhbr - Can you propose some specific text?

Reply:
This was updated after our conversation before the tech call on 5 Sept